pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -5318,21 +5832,32 @@ def _preprocess_data(model):
         roughness = model.df_parameters["Hydraulics"]["roughness"]
         max_head_loss = model.df_parameters["Hydraulics"]["max_head_loss"]
 
-        model.df_parameters["PipelineCapacityIncrements"] = {}
+        model.df_parameters["PipelineCapacityIncrements_Calculated"] = {}
         for key in model.df_parameters["PipelineDiameterValues"]:
-            diameter = model.df_parameters["PipelineDiameterValues"][key]
+            diameter = pyunits.convert_value(
+                model.df_parameters["PipelineDiameterValues"][key],
+                from_units=model.user_units["diameter"],
+                to_units=model.model_units["diameter"],
+            )
             flow_rate = (
                 (1 / 10.67) ** (1 / 1.852)
                 * roughness
                 * (max_head_loss**0.54)
                 * (diameter * 0.0254) ** 2.63
             )
 
-            # convert to bbl/week:
-            flow_rate *= 6.28981 * (3600 * 24 * 7)
+            # convert to volume/time:
+            days_in_period = model.decision_period / pyunits.days
+            # Make variable unitless
+            days_in_period = pyunits.convert(
+                days_in_period, to_units=pyunits.days / pyunits.days
```

Comment:
`days_in_period = model.decision_period / pyunits.days` results in 'week/days' rather than a values. The conversion to `pyunits.days / pyunits.days` results in a numerical value, in this case 7 because there are 7 days in a decision period.